index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="nav-container">
+      <div class="toolbar">
+        <div class="logo">
+          <p>Rose Okpe</p>
+        </div>
+        <nav>
+          <div>
+            <a class="nav-button" href="#"
+              ><img src="images/home-icon.png" alt=""
+            /></a>
+          </div>
+        </nav>
+      </div>
+    </div>
+
+    <!--    HEADLINE      -->
+    <div class="head-container">
+      <div class="headline">
+        <div class="primary-text">
+          <p>Rose Okpe</p>
+        </div>
+        <div class="support-text">
+          <p>Software Developer</p>
+          <p></p>
+        </div>
+
+        <div class="intro-text">
+          <p>
+            I can help you build a product,<br />feature or website look through
+            <br />some of my work and experience! if<br />you like what you see
+            and have a <br />
+            project you need coded, don't <br />
+            hestitate to contact me.
+          </p>
+        </div>
+        <hr />
+        <div class="frame38-container">
+          <div class="languages">
+            <h4>Languages</h4>
+            <div class="language-list">
+              <p>Javascript</p>
+              <p>Ruby</p>
+              <p>Html</p>
+              <p>Css</p>
+            </div>
+          </div>
+          <div class="frameworks">
+            <h4>Frameworks</h4>
+            <div class="framework-list">
+              <p>Bootsrap</p>
+              <p>Ruby on Rails</p>
+              <p>RSPec</p>
+              <p>CapyBara</p>
+              <p>Selenium</p>
+            </div>
+          </div>
+          <div class="skills">
+            <h4>Skills</h4>
+            <div class="list-skills">
+              <p>Codekit</p>
+              <p>Github</p>
+              <p>Codepen</p>
+              <p>Gitlab</p>
+              <p>Terminal</p>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] Let use `ul` tag and `li` to group this buttons.Check this [link ](https://www.w3schools.com/tags/tag_ul.asp)for `ul` tag

Reply:
@uwadonat thank you for the review but the last request not clear to me. You want me to use ul tag and li to the listed skills, frameworks and Languages but that will give these items bullet sign which are not in the design.